std_msgs/Header header

string radar_id

geometry_msgs/Twist radar_placement

float32 h_fov
float32 v_fov

RadarOutputTarget[] targets